JASP-Desktop/po/jasp_de.po

```diff
@@ -19873,9 +19855,10 @@ msgctxt "LDgamma|"
 msgid "Scale:"
 msgstr ""
 
+#, fuzzy
 msgctxt "LDgamma|"
 msgid "Rate:"
-msgstr ""
+msgstr "Rate:"
```

Comment:
@boutinb, @FransMeerhoff : So will filled in fuzzies be marked as translated on Weblate? Because then we'll want to empty them before pushing

Reply:
fuzzy strings get a 'Needs Action' warning in WebLate. You can also select all Needing action strings to check them in WebLate.

Reply:
Ah, neat! Good to hear it isn't too barebones